### DISCUSSION TITLE
repository search: why are repository descriptions excluded?

### DISCUSSION THREAD

**Discussion by nick-youngblut** [community/community#45591] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My organization has a large number of repos on GitHub. The repository search only looks for matches to the repository titles and does not also search the repo descriptions. Allowing for the search of repo descriptions would be a great feature to add.

**Comment by jge162** [comment]
@nick-youngblut GitHub does offer a way to search for repositories based on their descriptions, you can use the 'in:description' query parameter in the search bar. For example, if you want to search for repositories that mention "machine learning" in their descriptions, you can use the query 'machine learning in:description' This will return a list of repositories whose descriptions contain the phrase "machine learning".
I did searches in your GitHub below to show demo. 

<img width="698" alt="Screenshot 2023-01-27 at 8 25 34 PM" src="https://user-images.githubusercontent.com/31228460/215241861-c1a80f88-29bf-45b3-a50a-3ad7e3f81831.png">
<img width="861" alt="Screenshot 2023-01-27 at 8 21 46 PM" src="https://user-images.githubusercontent.com/31228460/215241763-8e8739fc-50f3-4f98-9f01-3aa433075f57.png">
<img width="1003" alt="Screenshot 2023-01-27 at 8 25 10 PM" src="https://user-images.githubusercontent.com/31228460/215241853-a83249d2-5424-4763-92b4-8284a8b40c0b.png">

**Reply by jge162** [reply]
Hi @nick-youngblut wanted to follow up and make sure you tested the search. Lmk if it works for you. Thanks.

**Comment by nick-youngblut** [comment]
Thank you @jge162 for the information! I now see the issue: I wasn't using full words for searching, and partial word matches do not return any hits. That's what I get for not spelling out long words or IDs.

**Reply by jge162** [reply]
Perfect. Thanks for the question. Have a great weekend.
